orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/RollingRedBlackStrategy.groovy

```diff
@@ -33,15 +34,13 @@ import org.springframework.context.ApplicationContextAware
 import org.springframework.stereotype.Component
 import static com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder.newStage
 
-import static com.netflix.spinnaker.orca.kato.pipeline.support.ResizeStrategySupport.*
-
 @Slf4j
 @Component
 class RollingRedBlackStrategy implements Strategy, ApplicationContextAware {
   final String name = "rollingredblack"
 
   @Autowired
-  DisableClusterStage disableClusterStage
+  DisableServerGroupStage disableServerGroupStage
```

Comment:
very elegant change, my mouth is agape in amazement

Reply:
i thought it was clever :)